Uchu.World/Objects/Components/Player/ModularBuilderComponent.cs

```diff
@@ -133,15 +133,17 @@ public async Task ConfirmFinish()
 
             var inventory = GameObject.GetComponent<InventoryManagerComponent>();
             
-            foreach (var temp in inventory[InventoryType.TemporaryModels].Items)
-            {
-                await inventory.MoveItemBetweenInventoriesAsync(
-                    temp,
-                    temp.Count,
-                    InventoryType.TemporaryModels,
-                    InventoryType.Models
-                );
-            }
+                foreach (var temp in inventory[InventoryType.TemporaryModels].Items)
+                if (temp.Count < 0) ; // Make sure we account for builds where no parts were used
```

Comment:
Is there a reason there is an empty `temp.Count < 0` condition with an `else` instead of just `temp.Count >= 0` with the condition.